packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -799,9 +835,150 @@ export const RemoteServerFactory = {
+            return Buffer.from(body, isBase64Encoded ? 'base64' : 'utf8')
+        }
+
+        const _serverlessExpressHandler = serverlessExpress({
+            app,
+            // the underlying serverless-express library strips `headers`
+            // which our ssr-infrastructure expects, these mappings restore
+            // and pass through all request / response properties
+            eventSource: {
+                getRequest: ({
+                    event,
+                    method = event.httpMethod,
+                    path = getPathWithQueryStringParams({event})
+                }) => {
+                    let headers = {}
+
+                    if (event.multiValueHeaders) {
+                        headers = getCommaDelimitedHeaders({
+                            headersMap: event.multiValueHeaders,
+                            lowerCaseKey: true
+                        })
+                    } else if (event.headers) {
+                        headers = event.headers
+                    }
+
+                    let body
+
+                    if (event.body) {
+                        body = getEventBody({event})
+                        const {isBase64Encoded} = event
+                        headers['content-length'] = Buffer.byteLength(
+                            body,
+                            isBase64Encoded ? 'base64' : 'utf8'
+                        )
+                    }
+
+                    const remoteAddress =
+                        (event &&
+                            event.requestContext &&
+                            event.requestContext.identity &&
+                            event.requestContext.identity.sourceIp) ||
+                        ''
+
+                    return {
+                        method,
+                        headers,
+                        body,
+                        remoteAddress,
+                        path
+                    }
+                },
+                getResponse: ({statusCode, body, headers, isBase64Encoded}) => {
+                    const multiValueHeaders = {}
+
+                    Object.entries(headers).forEach(([headerKey, headerValue]) => {
+                        const headerArray = Array.isArray(headerValue)
+                            ? headerValue.map(String)
+                            : [String(headerValue)]
+
+                        multiValueHeaders[headerKey.toLowerCase()] = headerArray
+                    })
+
+                    return {statusCode, body, headers, multiValueHeaders, isBase64Encoded}
+                }
+            },
+            // handler,
+            // binarySettings: {
+            //     isBinary: ({headers}) => true,
+            //     contentTypes: binaryMimeTypes,
+            //     contentEncodings: []
+            // },
+
+            // TODO: this is for debugging, remove
+            respondWithErrors: true
+            // resolutionMode: 'CALLBACK',
```

Comment:
@bfeister I believe `resolutionMode` still needs to be set as `CALLBACK`

You can see it is still a parameter expected by the library https://github.com/vendia/serverless-express/blob/mainline/src/configure.js#L25